[glib:arm64-linux] build failure

**Host Environment**
- OS: linux arm64 (raspi 4B)
- Compiler: gcc (Ubuntu 9.3.0-17ubuntu1~20.04)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install glib`

**Failure logs** 

```
# VCPKG_FORCE_SYSTEM_BINARIES=true /vcpkg/vcpkg install glib
Computing installation plan...
The following packages will be built and installed:
    glib[core]:arm64-linux -> 2.66.4
Detecting compiler hash for triplet arm64-linux...
Could not locate cached archive: /root/.cache/vcpkg/archives/35/35d8c167aa4e4b981bd31391cea68e503b8078fc.zip
Starting package 1/1: glib:arm64-linux
Building package glib[core]:arm64-linux...
-- Using community triplet arm64-linux. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: /vcpkg/triplets/community/arm64-linux.cmake
-- Using /vcpkg/downloads/glib-2.66.4.tar.xz
-- Cleaning sources at /vcpkg/buildtrees/glib/src/glib-2-b84205c786.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /vcpkg/downloads/glib-2.66.4.tar.xz
-- Applying patch use-libiconv-on-windows.patch
-- Applying patch fix-libintl-detection.patch
-- Using source at /vcpkg/buildtrees/glib/src/glib-2-b84205c786.clean
-- Configuring arm64-linux-dbg
-- Configuring arm64-linux-rel
-- Found external meson('0.58.1').
-- Found external ninja('1.10.2').
CMake Error at scripts/cmake/vcpkg_configure_meson.cmake:240 (message):
  Unhandled machine: aarch64

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_meson.cmake:380 (vcpkg_internal_meson_generate_cross_file)
  ports/glib/portfile.cmake:42 (vcpkg_configure_meson)
  scripts/ports.cmake:139 (include)


Error: Building package glib:arm64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: glib:arm64-linux
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
```


**Additional context**

It seems to be a problem caused by meson, it fails at `Unhandled machine: aarch64`.

If I change the `scripts/cmake/vcpkg_configure_meson.cmake` like this, the package can be installed successfully.

```diff
diff --git a/scripts/cmake/vcpkg_configure_meson.cmake b/scripts/cmake/vcpkg_configure_meson.cmake
index 5128422..c62a11f 100644
--- a/scripts/cmake/vcpkg_configure_meson.cmake
+++ b/scripts/cmake/vcpkg_configure_meson.cmake
@@ -220,7 +220,7 @@ function(vcpkg_internal_meson_generate_cross_file _additional_binaries) #https:/
             OUTPUT_VARIABLE MACHINE
             COMMAND_ERROR_IS_FATAL ANY)
 
-        if(MACHINE MATCHES "arm64")
+        if(MACHINE MATCHES "arm64|aarch64")
             set(BUILD_CPU_FAM aarch64)
             set(BUILD_CPU armv8)
         elseif(MACHINE MATCHES "x86_64|amd64")

```

Should I create a PR for this change? I'm really not sure of the possible side effects.

--- a/scripts/cmake/vcpkg_configure_meson.cmake
+++ b/scripts/cmake/vcpkg_configure_meson.cmake
@@ -241,7 +241,7 @@ function(z_vcpkg_meson_generate_cross_file additional_binaries) #https://mesonbu
         # Show real machine architecture to visually understand whether we are in a native Apple Silicon terminal or running under Rosetta emulation
         debug_message("Machine: ${MACHINE}")
 
-        if(MACHINE MATCHES "arm64")
+        if(MACHINE MATCHES "arm64|aarch64")
             set(build_cpu_fam aarch64)
             set(build_cpu armv8)
         elseif(MACHINE MATCHES "x86_64|amd64")

